collector/cpu_vulnerabilities_linux.go

```diff
@@ -0,0 +1,61 @@
+package collector
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs/sysfs"
+)
+
+import (
+	"github.com/go-kit/log"
+)
+
+type cpuVulnerabilitiesCollector struct {
+	vulnDesc        *prometheus.Desc
+	totalCntDesc    *prometheus.Desc
+	vulnerabilities map[string]*sysfs.Vulnerability
+	total           float64
+}
+
+func init() {
+	registerCollector("cpu_vulnerabilities", defaultDisabled, NewVulnerabilitySysfsCollector)
```

Comment:
Let's make this a const

```
const cpuVulerabilitiesCollector = "cpu_vulnerabilities"
```

This way it can be used for the subsystem in `prometheus.BuildFQName()`.